### ISSUE TITLE
TS2315 Type 'ActionType' is not generic

### ISSUE THREAD

**Issue by egor-xyz** [piotrwitek/typesafe-actions#131]
```JSON
{
  "typesafe-actions": "^3.2.1",
  "typescript": "^3.4.1",
}
```

After upgrade to react-scripts start to get error:
`TS2315 Type 'ActionType' is not generic`

```JavaScript
// actions.ts
export const open = createStandardAction('OPEN_DIALOG')<DialogType>();
export const close = createStandardAction('CLOSE_DIALOG')<DialogType>();
```

```JavaScript
// reducer.ts
import { ActionType } from 'typesafe-actions';
import * as actions from './actions';
export type EditorAction = ActionType<typeof actions>;
```

Result:
TS2315 Type 'ActionType' is not generic

**Comment by maximematheron** [comment]
I have the exact same error.
Did you find the fix?

**Comment by maximematheron** [comment]
Fixed by installing the following version: 3.3.0-rc

Check this issue: https://github.com/piotrwitek/typesafe-actions/issues/130
